### PR TITLE
Make "Install Chromium" CI step failable

### PR DESCRIPTION
**Pull request by thiagowfx**
Bug: #693

### CODE REVIEW

**Comment by thiagowfx** [comment]
`chromium@115` does not work. We need rev number.

**Comment by OrKoN** [comment]
@thiagowfx that's why you should use CfT ;-)

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Install and build npm dependencies
         run: npm ci
       - name: Install Chromium
-        run: echo "chromium_binary=$(npx @puppeteer/browsers install chromium@latest | cut -f 2- -d' ')" >> $GITHUB_ENV
+        run: echo "chromium_binary=$(npx @puppeteer/browsers install chromium@1141358 | cut -f 2- -d' ')" >> $GITHUB_ENV
```
Comment (OrKoN) [inline .github/workflows/e2e.yml:30]: let's add a file (e.g., `.browser`) to the root of the repository containing `chromium@1141358` and read that file in CI to get the pinned browser@version?

**Review by sadym-chromium — APPROVED** [review]
LGTM. Alex's suggestion would make it a bit smoother, but can be done later.

**Comment by thiagowfx** [comment]
Great: I made a change to make `install chromium` fail. Previously, it was always succeeding, because of the pipe. `-o pipefail` will make it fail now.

image

**Comment by thiagowfx** [comment]
Let me merge just one of the commits for now.

**Pull request merged**